### ISSUE TITLE
Taiyi-CLIP-Roberta-102M-Chinese Finetuning报错

### ISSUE THREAD

**Issue by 1306825592** [IDEA-CCNL/Fengshenbang-LM#452]
再Fengshenbang-LM/fengshen/examples/clip_finetune/clip_finetune_flickr.py中
计算相似性这里
打印出来的shape显示
![image](https://github.com/IDEA-CCNL/Fengshenbang-LM/assets/47473445/4b0a5b1c-8f7c-41ce-8845-9fe0a44badc2)
![image](https://github.com/IDEA-CCNL/Fengshenbang-LM/assets/47473445/4b0a5b1c-8f7c-41ce-8845-9fe0a44badc2)
logits_per_image = (logit_scale * image_features @ text_features.t()).detach().cpu()
logit_scale 为 torch.Size([5000, 512])  image_features torch.Size([5000, 2])  label 为5000

包括这里也有shape 不一致的情况
![image](https://github.com/IDEA-CCNL/Fengshenbang-LM/assets/47473445/fc2b630a-b929-4399-9b84-32a24f14c26f)

请问是我复现方式不对吗
